test/crypten/test_context.py

```diff
@@ -219,14 +220,46 @@ def test_context_plan_with_model(workers):
         [alice, bob], master_addr="127.0.0.1", model=pytorch_model, dummy_input=dummy_input
     )
     @sy.func2plan()
-    def plan_func_model(model=None, crypten=crypten):
+    def plan_func_model(model=None, crypten=crypten):  # noqa: F821
         t = crypten.load("crypten_data", 0)
 
-        model.encrypt()  # noqa: F821
-        out = model(t)  # noqa: F821
-        model.decrypt()  # noqa: F821
+        model.encrypt()
+        out = model(t)
+        model.decrypt()
         out = out.get_plain_text()
-        return model, out  # noqa: F821
+        return model, out
 
     result = plan_func_model()
     assert th.all(result[0][1] == result[1][1])
+
+
+def test_context_plan_with_model_private(workers):
```

Comment:
The model here compared to the previous test could be private (ie hidden) to the local worker?
If so you can add a line to explain the difference with the previous test!

Reply:
Done